### ISSUE TITLE
Error loading Python module when bedtools version has atypical name

### ISSUE THREAD

**Issue by dgroot** [daler/pybedtools#275]
My version of Bedtools is "bedtools v2.25.0-96-g5ee3285-dirty" (not sure why, this is what my cluster manager has installed). When I import pybedtools into Python, I get the following error:

```
>>> import pybedtools
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/genome/home/Petronis/dgroot/miniconda3/lib/python3.6/site-packages/pybedtools/__init__.py", line 17, in <module>
    from .bedtool import BedTool
  File "/genome/home/Petronis/dgroot/miniconda3/lib/python3.6/site-packages/pybedtools/bedtool.py", line 388, in <module>
    class BedTool(object):
  File "/genome/home/Petronis/dgroot/miniconda3/lib/python3.6/site-packages/pybedtools/bedtool.py", line 1637, in BedTool
    @_wraps(prog='bed12ToBed6', implicit='i', bam=None, other=None)
  File "/genome/home/Petronis/dgroot/miniconda3/lib/python3.6/site-packages/pybedtools/bedtool.py", line 167, in _wraps
    p = subprocess.Popen(helpers._version_2_15_plus_names(prog) + ['-h'],
  File "/genome/home/Petronis/dgroot/miniconda3/lib/python3.6/site-packages/pybedtools/helpers.py", line 287, in _version_2_15_plus_names
    _check_for_bedtools()
  File "/genome/home/Petronis/dgroot/miniconda3/lib/python3.6/site-packages/pybedtools/helpers.py", line 89, in _check_for_bedtools
    settings.bedtools_version = [int(i) for i in vv.split(".")]
  File "/genome/home/Petronis/dgroot/miniconda3/lib/python3.6/site-packages/pybedtools/helpers.py", line 89, in <listcomp>
    settings.bedtools_version = [int(i) for i in vv.split(".")]
ValueError: invalid literal for int() with base 10: '0-96-g5ee3285-dirty\n'
```

I think pybedtools is expecting an integer version number but the actual name contains text.

**Comment by timchu90** [comment]
had the same problem.
fixed it by editing [PATH]/pybedtools/helpers.py

and adding:

vv = vv.split('-')[0]

after : 
vv = v.decode().split('v')[1]

in the function _check_for_bedtools()

**Comment by daler** [comment]
Thanks for reporting, fixed in #281
